### PR TITLE
e2e: use translation keys instead of strings

### PR DESCRIPTION
**Pull request by ncomont**
- translation keys are automatically prefixed by the client id (`banking`, `onboarding`, or `shared`)
- a new script helps to automatically replace translated strings by their translation keys (`yarn match-e2e-translations`)

### CODE REVIEW

File: tests/1-company.onboarding.ts [hunk 1]
```diff
@@ -118,19 +118,19 @@ test("French company onboarding", async ({ browser, page, request }) => {
...
 
   await page.getByLabel(t("onboarding.company.step.organisation2.monthlyPaymentLabel")).click();
-  await page.getByText("More than €100,000").click();
+  await page.getByText(t("shared.monthlyPaymentVolume.moreThan100000")).click();
 
   await page.getByRole("button", { name: t("onboarding.common.next") }).click();
 
   const nicolasBenadyTile = page.locator("section", { hasText: "nicolas benady" });
-  const nicolasSaisonTile = page.locator("section", { hasText: "nicolas, rene, michel saison" });
+  const nicolasSaisonTile = page.locator("section", { hasText: "nicolas, rené, michel saison" });
```
Comment (ncomont) [inline tests/1-company.onboarding.ts:133]: cleaner fix to come in another pr

File: tests/utils/i18n.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { createIntl, createIntlCache } from "@formatjs/intl";
+import sharedEN from "@swan-io/shared-business/src/locales/en.json";
+import bankingEN from "../../clients/banking/src/locales/en.json";
+import onboardingEN from "../../clients/onboarding/src/locales/en.json";
```
Comment (bloodyowl) [inline tests/utils/i18n.ts:4]: don't we have a risk of key collision here? we could use the `t` functions from each app directly in the tests to make sure there's no risk

Comment (ncomont) [reply tests/utils/i18n.ts:4]: every bundle is prefixed by the client key on line 9 to avoid that

Comment (bloodyowl) [reply tests/utils/i18n.ts:4]: oh right, missed that bit

**Pull request merged**